### ISSUE TITLE
Error after upgrading to 7.4.5: dyld: Library not loaded

### ISSUE THREAD

**Issue by schliflo** [henkrehorst/homebrew-php#52]
I just installed the new version and am getting the following error:

```
dyld: Library not loaded: /usr/local/opt/libffi/lib/libffi.6.dylib
  Referenced from: /usr/local/bin/php
  Reason: image not found
```

**Comment by schliflo** [comment]
seems like `libffi` got updated and the php executeable expects to find it at `/usr/local/opt/libffi/lib/libffi.6.dylib` which isn't there any more.
I symlinked `/usr/local/opt/libffi/lib/libffi.6.dylib` to `/usr/local/opt/libffi/lib/libffi.dylib` which in turn seems to be symlinked to the current version.
```
cd /usr/local/opt/libffi/lib
ln -s libffi.dylib libffi.6.dylib
```
This fixed it for me.

I don't know if this is an issue particular to the php build provided by this repo. Feel free to close the issue if this is unrelated.
